Sources/SourceKitLSP/Swift/SwiftTestingScanner.swift

```diff
@@ -95,7 +95,10 @@ struct TestingAttributeData {
           return false
         }
       }.flatMap(\.arguments)
-      .compactMap { $0.expression.as(StringLiteralExprSyntax.self)?.representedLiteralValue }
+      .compactMap {
+          $0.expression.as(StringLiteralExprSyntax.self)?.representedLiteralValue ??
+          $0.expression.as(MemberAccessExprSyntax.self)?.declName.baseName.text
```

Comment:
Does this correctly handle symbolic tags specified as:

- `Testing.Tag.foo`
- `Tag.foo`
- `Tag.SomeNestedType.foo`

?

Reply:
All three of these evaluate to `foo`. Is that how you'd expect it? I figured tags don't really have a hierarchy, just a name. However it does open it up to conflicts if you have `Tag.Foo.foo` and `Tag.Bar.foo`.

Reply:
The first two are equivalent to `.foo`. The last is equivalent to `.SomeNestedType.foo`, and that's a distinct tag as far as swift-testing is concerned.

Reply:
I think in the end, we want the string value for the tag, no? I think that's hard to find using SwiftSyntax.

Reply:
I've updated to take in to account nested types. Let me know if there is a more elegant way to do it; string manipulation is probably not the best. I think we do know the type will always start with `Tag.` when nested.

Reply:
I think it would be good to add a test case for `Tag.SomeNestedType.foo`.